packages/@aws-cdk/aws-ecs-patterns/lib/base/queue-processing-service-base.ts

```diff
@@ -86,6 +86,14 @@ export interface QueueProcessingServiceBaseProps {
    */
   readonly queue?: IQueue;
 
+  /**
+   * The maximum number of times that a message can be received by consumers.
+   * When this value is exceeded for a message the message will be automatically sent to the Dead Letter Queue.
+   *
+   * @default 5
```

Comment:
Curious as to why you default to 5?

Reply:
it is a magic number, it is however configurable by user.